docs/index.rst

```diff
@@ -0,0 +1,30 @@
+Welcome to SymbiFlow examples!
+==============================
+
+This guide explains how to get started with SymbiFlow and build example designs from the `SymbiFlow Examples <https://github.com/antmicro/symbiflow-examples>`_ GitHub repository. It currently focuses on two FPGA families:
```

Comment:
```suggestion
This guide explains how to get started with SymbiFlow and build example designs from the `SymbiFlow Examples <https://github.com/SymbiFlow/symbiflow-examples>`_ GitHub repository. It currently focuses on following FPGA families:
```

Fixed the URL and removed the `two` (that way when someone adds one more family, they don't have to change this).